source/draw/README.md

```diff
@@ -53,6 +53,8 @@ Using destination or transform on the CPU would require a rewrite of legacy draw
 | inputImage | X | X | X |
 | viewport | X | X |  |
 | source | X | X |  |
+| flipSourceX | X | X |  |
+| flipSourceY | X | X |  |
```

Comment:
Using source flipping in text writing would only mess up the characters but for drawing quads, it is implemented on both CPU and GPU.